yarn.lock

```diff
@@ -2287,6 +2287,20 @@
     xcode "^3.0.0"
     xml-js "^1.6.11"
 
+"@expo/configure-splash-screen@0.4.0":
```

Comment:
I'm not sure why it hasn't removed the `"@expo/configure-splash-screen@0.3.4"` that is still listed above 🤔 
I think there should be only one entry for this module as AFAIK it is only used once as a dependency in this monorepo 🤔

Reply:
it's also depended on by expo/config and expo/config-plugins